code/datums/elements/caltrop.dm

```diff
@@ -96,7 +96,10 @@
 	H.apply_damage(damage, BRUTE, picked_def_zone, wound_bonus = CANT_WOUND)
 
 	if(!(flags & CALTROP_NOSTUN)) // Won't set off the paralysis.
-		H.Paralyze(60)
+		if(H.dna.species.id == "squid")
+			H.Paralyze(90)
+
+		else H.Paralyze(60)
```

Comment:
use is_species instead, preferably add a helper here: https://github.com/The-Merchants-Guild/Merchant-Station-13/blob/master/code/__DEFINES/is_helpers.dm#L59